tests/optimize/test_spsa.py

```diff
@@ -0,0 +1,466 @@
+# Copyright 2018-2022 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for the SPSA optimizer"""
+import os
+
+os.environ["OMP_NUM_THREADS"] = "2"
```

Comment:
Why was this added?

Reply:
I was testing the parallelization options, but is not necessarily.